packages/devtools/src/commands/deleteCookie.js

```diff
@@ -0,0 +1,11 @@
+export default async function deleteCookie ({ name }) {
+    const page = this.windows.get(this.currentWindowHandle)
+    const cookies = await page.cookies()
+    const cookieToDelete = cookies.find((cookie) => cookie.name === name)
+
+    if (cookieToDelete) {
```

Comment:
should we throw if cookie is not found?

Reply:
no, there is nothing that explicitly says that in the protocol